README.md

```diff
@@ -57,19 +30,26 @@ A Gutter Game is when the player never hits a pin (20 zero scores).
 
 ### Perfect Game
 
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
-
-In the image below you can find some score examples.
-
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
+A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.  
 
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+## Getting Started
 
-## Code Review
+This program runs in the terminal console allowing for user input.
 
-In code review we'll be hoping to see:
+To begin,clone the repository from GitHub:
```

Comment:
Nice! Including installation instructions and instructions to run the code for the reviewer is an awesome practice to have